docs/.env.example

```diff
@@ -0,0 +1,27 @@
+# Configuration for the database
+## Where is the database hosted
+POSTGRES_HOST=localhost
+## On which port is the database reachable
+POSTGRES_PORT=5432
+## What is the name of the database to use
+POSTGRES_DB=postgres
+## Which username should the application use to connect to the database
+POSTGRES_USER=postgres
+## What is the password of the user to connect to the database
+POSTGRES_PASSWORD=postgres-s3cr3t
+
+# Configuration for the GitHub App
+## What is the ID of the GitHub App
+GITHUB_APP_ID=token_example
+## Where is the private key to be used to authenticate to the GitHub App
+GITHUB_APP_PRIVATE_KEY_PATH=path-to-private-key
+## The name of the organization or user where the GitHub App was installed
+GITHUB_APP_INSTALLATION_NAME=organization-or-username
+
+# Configuration of the application schedulers
+## Frequency the update-center content is fetched
+## This means on the hour, every 2 hours
+UPDATE_CENTER_CRON=0 0 */2 * * *
+## Frequency the probes and scoring processed are executed
+## This means at 5, every 2 hours
```

Comment:
This is the line that confused me. Does this mean at 5pm, 5am or 5 minutes past the top the hour, every two hours?

Reply:
I see. It's "past 5, every hour", so 2:05am, 4:05am, etc. How would you phrase that?

Reply:
perhaps @kmartens27 could chime in?

Reply:
I like what @jleon33 shared as his last suggestion. I was thinking of something like
"This means every two hours, five minutes past the top of the hour"

Reply:
Thank you both. I integrated your suggestion in https://github.com/jenkins-infra/plugin-health-scoring/pull/203/commits/ade5d8a8116c9c4025cfdc7d89b7abd7e5b75906